recipes/emsarray/meta.yaml

```
@@ -0,0 +1,76 @@
+{% set name = "emsarray" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/emsarray-{{ version }}.tar.gz
+  sha256: 63c92c95483ec263aecf88266acf102513e2ce74099c035172affe4eff959729
+
+build:
+  number: 0
+
+outputs:
+  - name: emsarray-core
+    build:
+      noarch: python
+      script: python -m pip install . -vv --no-deps
```

```suggestion
      script: python -m pip install . -vv --no-deps
```
This should be moved into a separate file, then it should work.

Example: https://github.com/conda-forge/sagemaker-inference-toolkit-feedstock/tree/main/recipe